pkg/control/control.go

```diff
@@ -158,22 +158,22 @@ func (c *Controller) syncExec(f func(i int)) {
 func (c *Controller) setUpDB() {
 	log.Printf("begin to set up database")
 	c.syncExec(func(i int) {
```

Comment:
`syncExec` will trigger input function for every pod, includes tikv, pd, tidb, we should set up db for tidb pod only here. A further problem is that all tidb pods are under a same nodeport service which runs load balance, we should not set up db for the same db address multiple times.